eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -145,10 +151,15 @@ stages:
                 runOnce:
                   deploy:
                     steps:
-                      - checkout: self
-                        path: azure-sdk-for-java
                       - checkout: azure-sdk-build-tools
                         path: azure-sdk-build-tools
+                      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+                        parameters:
+                          SkipDefaultCheckout: true
+                          Repositories:
+                            - Name: Azure/azure-sdk-for-java
```

Comment:
Why do we need to pass all these extra parameters in this case? Is it because this job is mult-checkout?

Reply:
The default behavior when Repositories aren't specified mirrors the devops default behavior, which is to clone into the working directory `/1/s`. When multiple repositories are specified, we need to clone each to a separate path, and this is how I set up the parameters for the sparse checkout job to do that.